main/.vitepress/themeConfig/nav.js

```diff
@@ -38,6 +38,14 @@ export const nav = [
         ariaLabel: 'Agoric CLI',
         link: '/guides/agoric-cli/',
       },
+      {
+        text: 'Agoric API Reference',
```

Comment:
I think we usually include SDK in the name for this stuff:

```suggestion
        text: 'Agoric SDK API Reference',
```